lib/matplotlib/collections.py

```diff
@@ -1262,62 +1262,60 @@ def __init__(self,
                  **kwargs
                  ):
         """
-        *positions*
-            a sequence of numerical values or a 1D numpy array.  Can be None
-
-        *orientation* [ 'horizontal' | 'vertical' | None ]
-            defaults to 'horizontal' if not specified or None
+        Parameters
+        ----------
+        positions : 1D array-like object.
+            Each value is an event.
 
-        *lineoffset*
-            a single numerical value, corresponding to the offset of the center
-            of the markers from the origin
+        orientation : [ None (default) | 'horizontal' | 'vertical'], optional.
+            The orientation of the **collection** (the event bars are along
+            the orthogonal direction). Defaults to 'horizontal' if not
+            specified or None.
 
-        *linelength*
-            a single numerical value, corresponding to the total height of the
-            marker (i.e. the marker stretches from lineoffset+linelength/2 to
-            lineoffset-linelength/2).  Defaults to 1
+        lineoffset : scalar, optional. Default is 0.
+            The offset of the center of the markers from the origin.
 
-        *linewidth*
-            a single numerical value
+        linelength : scalar, optional. Default is 1.
+            The total height of the marker (i.e. the marker stretches from
+            ``lineoffset + linelength/2`` to ``lineoffset - linelength/2``).
 
-        *color*
-            must be a sequence of RGBA tuples (e.g., arbitrary color
-            strings, etc, not allowed).
+        linewidth : scalar or None, optional. Default is None.
+            If it is None, defaults to its rcParams setting, in sequence form.
 
-        *linestyle* [ 'solid' | 'dashed' | 'dashdot' | 'dotted' ]
+        color : color, sequence of colors or None, optional. Default is None.
+            If it is None, defaults to its rcParams setting, in sequence form.
 
-        *antialiased*
-            1 or 2
+        linestyle : str or tuple, optional. Default is 'solid'.
+            Valid strings are [ 'solid' | 'dashed' | 'dashdot' | 'dotted' |
+            '-' | '--' | '-.' | ':' ]. Dash tuples have to be of the form::
```

Comment:
@QuLogic I am not sure I understand what you mean. You would write something like:
```
        linestyle : str or tuple, optional. Default is 'solid'.
            Valid strings are ['solid', 'dashed', 'dashdot', 'dotted',
            '-', '--', '-.', ':']. Dash tuples should be of the form::
```
or
```
        linestyle : ['solid', 'dashed', 'dashdot', 'dotted', '-', '--', '-.',
                     ':', tuple], optional
            Dash tuples should be of the form::

```
instead?